auth_token/contrib/common/default/views.py

```diff
@@ -60,11 +60,10 @@ class TokenLogoutView(LogoutView):
     @method_decorator(never_cache)
     def dispatch(self, request, *args, **kwargs):
         logout(request)
-        next_page = self.get_next_page()
-        if next_page:
+        if self.next_page:
```

Comment:
`get_next_page` was renamed to `get_success_url` in django 4.1, shouldn't you use the `get_success_url` method?

Reply:
good catch :-) ten wrapper řeší aji redirecty

Reply:
I removed it all together.
That logic was removed from dispatch method completely in Django. Compare https://github.com/django/django/blob/4.0.10/django/contrib/auth/views.py#L126 with https://github.com/django/django/blob/4.1.13/django/contrib/auth/views.py#L140

@Formulka please have a look

Reply:
I see, thank you, yes this is ok 👍